src/cf-layout/src/cf-layout.less

```diff
@@ -257,18 +255,22 @@
 .content_intro,
 .content_main,
 .content_sidebar {
-    padding: unit( @grid_gutter-width / @base-font-size-px, em )
-             unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+    padding:
+        unit( @grid_gutter-width / @base-font-size-px, em )
+        unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
 
     .respond-to-min( @bp-sm-min, {
         .grid_column( 12 );
-        padding: unit( ( @grid_gutter-width * 1.5 ) / @base-font-size-px, em )
-                 unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+
+        padding:
+        unit( ( @grid_gutter-width * 1.5 ) / @base-font-size-px, em )
```

Comment:
The apparent missing indent here and in the one below appears to be a shortcoming with stylelint, as it's trying to align it with padding options on L:259-260. It appears the respond-to-min mixin is throwing it off. Do we care? While this is slightly unfortunate, the other things the linter fixes in this PR outweighs the two rules minor indenting inconsistency here.

Reply:
Depending on what our linting posture will be going forward, we could maybe revert the auto-fix in these cases. If we get everything known-good with this PR, could we then only lint changed lines going forward? Another option could be to not use the shorthand property here.

Reply:
Autofixing will look at all files, so if we submitted this known-good the next time autofixing was run it would move these lines again. I tried stylelint code comments to disable the linting for this section (`/* stylelint-disable */` `/* stylelint-enable */`), but it inexplicably ignores it. This seems to be a known issue, but not repeatable by the maintainers https://github.com/stylelint/stylelint/issues/456 
I looked at breaking out the shorthand here, but it does add a lot of extra lines here, so I'd lean toward living with the indenting here and revisiting in a future stylelint update and see if it can be ignored. Happy to go the breakout route though if that's preferable to the (mis)indent.